pennylane/pulse/parametrized_evolution.py

```diff
@@ -505,6 +505,65 @@ def fun(y, t):
             mat = mat[-1]
         return qml.math.expand_matrix(mat, wires=self.wires, wire_order=wire_order)
 
+    def label(self, decimals=None, base_label=None, cache=None):
+        r"""A customizable string representation of the operator.
+
+        Args:
+            decimals=None (int): If ``None``, no parameters are included. Else,
+                specifies how to round the parameters.
+            base_label=None (str): overwrite the non-parameter component of the label
+            cache=None (dict): dictionary that carries information between label calls
+                in the same drawing
+
+        Returns:
+            str: label to use in drawings
+
+        **Example:**
+
+        >>> H = qml.PauliX(1) + qml.pulse.constant * qml.PauliY(0) + jnp.polyval * qml.PauliY(1)
+        >>> params = [0.2, [1, 2, 3]]
+        >>> op = qml.evolve(H)(params, t=2)
+
+        >>> op.label()
+        "ParametrizedEvolution"
+        >>> op.label(decimals=2)
+        "ParametrizedEvolution\n(p=[0.20,[1.00,2.00,3.00]], t=[0. 2.])"
+        >>> op.label(base_label="my_label")
+        "my_label"
+        >>> op.label(decimals=2, base_label="my_label")
+        "my_label\n(p=[0.20,[1.00,2.00,3.00]], t=[0. 2.])"
+        """
+        op_label = base_label or self.__class__.__name__
```

Comment:
```suggestion
        op_label = base_label or "Parametrized\nEvolution"
```

![Screenshot 2023-08-22 at 5 29 13 PM](https://github.com/PennyLaneAI/pennylane/assets/6364575/591ab727-147a-4fbb-b37d-e413830f6427)

![Screenshot 2023-08-22 at 5 36 41 PM](https://github.com/PennyLaneAI/pennylane/assets/6364575/d4debf7d-2595-4078-b827-d4cf3eb0922b)

Text drawer removes all new lines, so not effected by adding in the `\n`.

Reply:
Good point, thanks!